agent/lib/auth_service.js

```diff
@@ -46,6 +46,7 @@ function authenticate(credentials, options) {
         var handleFailure = function (context) {
             if (!handled) {
                 handled = true;
+            	log.error('Authentication failed! error no: '+ context.error.errno + ' syscall: '+context.error.syscall+ ' address: '+context.error.address+ ' port: '+ context.error.port + ' code: '+ context.error.code);
```

Comment:
Minor nitpick, looks like indentation is a bit off?

Reply:
fixed my javascript indenting.